### PR TITLE
📝 Include a much simpler TypeScript example

### PR DESCRIPTION
**Pull request by omerose**
Changes: Docs update for Typescript-Cypress

Reasoning:
There was an overload of information in these docs (and example recipes)
and it took me several hours to set-up something that was relatively
simple if it was communicated better. Feel free to fork the repo I
put in the links. Please put something this simple in these docs.

### Translations updated

Changes made to documentation were also copied over to other languages (copying English text is fine).

- [ ] Japanese docs in [`/source/ja`](/source/ja).
- [ ] Chinese docs in [`/source/zh-cn`](/source/zh-cn).
- [ ] Not applicable (this is not a change to an `en` doc content).

### CODE REVIEW

File: source/guides/tooling/typescript-support.md [hunk 1]
```diff
@@ -40,7 +41,10 @@ The `"types"` will tell the TypeScript compiler to only include type definitions
 
 ## Types for custom commands
 
-When adding custom commands to the `cy` object, you can add their types to avoid TypeScript errors. You can find an example in the {% url "cypress-example-todomvc custom commands" https://github.com/cypress-io/cypress-example-todomvc#custom-commands %} repo.
+When adding custom commands to the `cy` object, you can add their types to avoid TypeScript errors. You can find the simplest implementation of Cypress and typescript in this {% url "repo example here" https://github.com/omerose/cypress-support %}.
```
Comment (jennifer-shehane) [inline source/guides/tooling/typescript-support.md]: Error from our text linter. You can run `npm run md-lint` in the future to see these errors before commiting.
>/root/cypress-documentation/source/guides/tooling/typescript-support.md
  44:156  ✓ error  Incorrect usage of the term: “typescript”, use “TypeScript” instead  terminology

Comment (omerose) [reply source/guides/tooling/typescript-support.md]: Made the change

**Comment by bahmutov** [comment]
Good idea @omerose but I looked at your repo and "google" is undefined for example - did you mean to have it as a function to be added as a command under the name "google" first?

Screen Shot 2019-06-17 at 9 42 01 AM

I do like the way you add definitions to the global cypress namespace, it is simpler than having a separate index.d.ts file

I would probably add links in the README of the example project to point at the support file and at the plugins file for people to quickly jump there.

Overall I think this is a good change @jennifer-shehane and makes sense to merge

**Comment by bahmutov** [comment]
Oh, and @omerose for testing dependencies you can either
- do it yourself by using http://github.com/bahmutov/next-update
- automate it using RenovateBot or some other dependency service, that's how we keep our example repos up to date https://glebbahmutov.com/blog/renovate-app/

**Pull request merged**